parcels/collection/collections.py

```diff
@@ -700,16 +797,22 @@ def merge(self, same_class=None):
 
         The function shall return the merged ParticleCollection.
         """
-        return None
+        if same_class is None:
+            return
+        if type(same_class) is type(self):
+            self.merge_same(same_class)
+        elif isinstance(same_class, ParticleCollection):
+            self.merge_collection(same_class)
 
     @abstractmethod
-    def split(self, indices=None):
+    def split(self, subset=None):
         """
         This function splits this collection into two disect equi-structured collections. The reason for it can, for
         example, be that the set exceeds a pre-defined maximum number of elements, which for performance reasons
         mandates a split.
 
-        On the other hand, this function can also internally split individual particles that are tagged byt status as
+        TODO - RETHINK IF TAHT IS STILL THE WAY TO GO:
```

Comment:
This either needs to be implemented before the PR merge, or expanded/explained better for follow-up